.gitattributes

```diff
@@ -0,0 +1,9 @@
+bin export-ignore
+stubs export-ignore
+tests export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+composer.lock export-ignore
+phpcs.xml.dist export-ignore
+phpunit.xml.dist export-ignore
+psalm.xml.dist export-ignore
```

Comment:
Adds some comments to explain what this file does, aligns the properties for easier reading, and does LF normalization.

```suggestion
# Exclude these files from release archives.
# This will also make them unavailable when using Composer with `--prefer-dist`.
# If you develop this package using Composer, use `--prefer-source`.
# https://blog.madewithlove.be/post/gitattributes/

bin              export-ignore
stubs            export-ignore
tests            export-ignore
.gitattributes   export-ignore
.gitignore       export-ignore
composer.lock    export-ignore
phpcs.xml.dist   export-ignore
phpunit.xml.dist export-ignore
psalm.xml.dist   export-ignore

# Auto detect text files and perform LF normalization
# https://pablorsk.medium.com/be-a-git-ninja-the-gitattributes-file-e58c07c9e915

* text=auto

# The above will handle all files NOT found below

*.md  text
*.php text
*.inc text
```